channels/al.m3u

```diff
@@ -1,6 +1,6 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/YaOhPf6.jpg" group-title="",ABC News
-http://176.31.74.164:8081/abcnewslive/abcnews/chunks.m3u8
+http://176.31.74.164:8081/abcnewslive/abcnews/playlist.m3u8
```

Comment:
New link does not work, original link works.

Reply:
Hi, this one also works fine in VLC & via streamtest https://streamtest.in/logs/http-176-31-74-164-8081-abcnewslive-abcnews-playlist-m3u8